### PR TITLE
[FTR] Dockerize application

### PR DESCRIPTION
**Pull request by dimeskigj**
## Description

Dockerized the app and added a docker-compose.yml file for easy deployment. 

 None of the .env variables are required, and the port is defaulted to 3000.  We'll need them when we implement incidents.
 
 Updated the README.md file to better reflect our fork.

## Testing

Run `docker-compose up`, go to localhost:305 in your browser.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+COPY . . 
+
+RUN mkdir -p /app/static/kener
+ENV PUBLIC_KENER_FOLDER=/app/static/kener
+
+RUN npm install
+RUN npm run build
+
+FROM node:21-alpine
```
Comment (dimitrievljupce) [inline Dockerfile:13]: Redundant I guess, it's the same image as the base one

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+version: "3.8"
+
+services:
+  kener-app:
+    restart: always
+    build:
+      context: .
+      dockerfile: Dockerfile
```
Comment (dimitrievljupce) [inline docker-compose.yml]: consider replacing this with certain image instead of referencing the Dockerfile

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    ports:
+      - "305:3000"
+    volumes:
+      - /volumes/kerner:/app/static/kerner
```
Comment (dimitrievljupce) [inline docker-compose.yml]: it's kerner for me :) change the name here

Comment (dimitrievljupce) [reply docker-compose.yml]: And also I needed to change this like the following 
`./kener:/app/static/kerner`
in order to make it work on mac

**Comment by dimitrievljupce** [comment]
Rename the `docker-compose.yaml` file to `compose.yaml`.

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+WORKDIR /app
```
Comment (dimitrievljupce) [inline Dockerfile:15]: WORKDIR is already set to /app

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+COPY prod.js ./
+COPY ./scripts ./scripts
+COPY ./config ./config
+
+COPY --from=BUILD /app/package*.json ./
+COPY --from=BUILD /app/build ./build
+
+RUN mkdir -p /app/static/kener
+ENV PUBLIC_KENER_FOLDER=/app/static/kener
```
Comment (dimitrievljupce) [inline Dockerfile:25]: You already have this one on line 8

File: docker-compose.yml (hunk 3 above)
Comment (dimeskigj) [reply docker-compose.yml]: Added a volume definition

File: docker-compose.yml (hunk 2 above)
Comment (dimeskigj) [reply docker-compose.yml]: when we setup CI

**Review by iboshkov — APPROVED** [review]

**Review by dimitrievljupce — APPROVED** [review]

**Pull request merged**